ext/fetch/lib.rs

```diff
@@ -519,37 +511,29 @@ impl HttpClientResource {
   }
 }
 
-#[derive(Deserialize, Debug)]
-#[serde(rename_all = "camelCase")]
-pub struct CreateHttpClientOptions {
```

Comment:
Not too convinced we should flatten this struct of options, since it could conceivably be extended and it's not on a hot path

Reply:
reverted